packages/sit-onyx/src/components/OnyxListbox/types.ts

```diff
@@ -1,24 +1,22 @@
 import type { SelectionOptionValue } from "../OnyxRadioButton/types";
 
 export type OnyxListboxProps<TValue extends SelectionOptionValue = SelectionOptionValue> = {
+  /**
+   * Aria label. Must be set for accessibility reasons.
+   */
+  label: string;
```

Comment:
will the listbox ever be used standalone without a select?
if not, the label won't be needed here I think because from a11y perspective the select above will provide all context you need. might get redundant else.
but you can keep it for now and we'll see how it works out when baking the Listbox into the Select.

Reply:
Yes in general the listbox can be used standalone :)
Also accessibility tests will fail if it does not have a aria label